### ISSUE TITLE
comprehensive test set

### ISSUE THREAD

**Issue by xzackli** [xzackli/WignerFamilies.jl#3]
This package needs a comprehensive test set which compares to either WignerSymbols.jl or the orthogonality relations, possibly both.

**Comment by hsgg** [comment]
I made a test for all the low j comparing with WignerSymbols.jl, see aa55aef8496a771d6d4ec6567347101db64e7d6b . Unfortunately, some tests fail. Would be interested in a PR?

**Comment by xzackli** [comment]
Hey @hsgg, I'd be interested in a PR for more tests! I'll also take a look at which tests failed.

**Comment by xzackli** [comment]
@hsgg thank you for the PR, I've now merged it! I had paused the project associated with this package, and now I finally am back to looking at this. 

Your tests enabled me to fix bugs revealed by the failing test, which I am extremely grateful for. I think this test set convinces me that this package is ready for a 1.0 release!

For future reference (mostly for me), there were two problems revealed by the test.
1. In the case where there are only *two* nontrivial elements in a Wigner 3j family, one obviously cannot apply a three-element recursion. Had to put in this case.
2. I didn't correctly handle the boundary element between classical and nonclassical regions.
